### DISCUSSION TITLE
Transcript pooling (video call setting)

### DISCUSSION THREAD

**Discussion by yehiaabdelm** [deepgram/community#427] [General help] [answered]
Is it possible to pool transcripts? For example, let's say I have 3 people on a call on our platform. Is it possible to send all three audio sources, but have one singular transcript for all three?

**Comment by jpvajda** [comment, marked as answer]
Hi @yehiaabdelm  Yes, it is possible to pool transcripts for multiple audio sources. Deepgram offers both diarization and multichannel ASR features that can be used to achieve this. Diarization focuses on separating an audio stream into segments according to speaker identity, while multichannel ASR combines multiple audio sources into a single transcript. 

The choice between diarization and multichannel ASR depends on the specific requirements of your audio and how you plan to integrate the transcripts into your platform. 

[This Guide](https://developers.deepgram.com/docs/multichannel-vs-diarization) explains both these features in detail  and should help.
